### PR TITLE
Update chapter "Evaluation 2"

### PR DESCRIPTION
**Pull request by ludelafo**

### CODE REVIEW

**Comment by ludelafo** [comment]
I'm thinking of doing one evaluation on paper to prepare the students for the exam.

Out of the two remaining evaluations (2 and 3), which format do you think would be the best for a paper evaluation?

- The second one (this one) so it makes 1 digital, 1 paper, 1 digital and the final exam paper?
- The third one so it makes 1 digital, 1 digital, 1 paper and the final exam paper?

**Comment by GeraudSilvestri** [comment]
> I'm thinking of doing one evaluation on paper to prepare the students for the exam.
> 
> Out of the two remaining evaluations (2 and 3), which format do you think would be the best for a paper evaluation?
> 
> * The second one (this one) so it makes 1 digital, 1 paper, 1 digital and the final exam paper?
> * The third one so it makes 1 digital, 1 digital, 1 paper and the final exam paper?

I'd say that making the 3rd one on paper is more interesting for them as it is closer to the final exam.
So they don't have paper then digital again, which could be misleading/confusing for the format of the final exam. By making the 3rd one paper, they are getting ready for the final that would be in the same format.

**Comment by ludelafo** [comment]
Good point.

The only thing I see is that the second part might contain some diagrams to draw. Making it on paper would be easier for the students.

**Comment by hadrylouis** [comment]
I agree with both your points of view. 

However, I think it would be more practical to do the evaluation 2 on paper, given the type of questions that are likely to be asked.

**Comment by ludelafo** [comment]
I'll update this chapter as soon a possible to reflect this.

File: 17-evaluation-2/PRESENTATION.md [hunk 1]
```diff
@@ -68,43 +67,94 @@ _paginate: false
...
+![bg right:40%][illustration]
+
+## Notes
 
-You can only have access to your own notes in paper format. Your notes must fit
-on one sheet of paper (recto). They can contain anything useful to you:
-definitions, schemas, code snippets, etc. You will be able to use them during
-the exam at the end of the semester.
+You have access to your own notes in paper format. Your notes must fit on **2
+(1 + 1.5) sheets of paper (recto)**.
```
Comment (hadrylouis) [inline 17-evaluation-2/PRESENTATION.md:126]: I am not sure I understand this

File: 25-evaluation-3/PRESENTATION.md [hunk 2]
```diff
@@ -121,8 +121,8 @@ solve a specific problem.
 
 ## Notes
 
-You have access to your own notes in paper format. Your notes must fit on **four
-sheets of paper (recto)**.
+You have access to your own notes in paper format. Your notes must fit on **4
+(1 + 1.5 + 1.5) sheets of paper (recto)**.
```
Comment (hadrylouis) [inline 25-evaluation-3/PRESENTATION.md:125]: I am not sure I understand this

File: 17-evaluation-2/PRESENTATION.md (hunk 1 above)
Comment (ludelafo) [reply 17-evaluation-2/PRESENTATION.md:126]: I wanted to illustrate that the students can use 2.5 sheets of paper (the `2` need to be updated, that's a mistake on my side), using the 1 previous note sheet from evaluation 1 + 1.5 new sheets of notes for the new evaluation.

I don't know if there is another way less confusing or do I just drop that part and only leave 2.5 sheets of paper and that's it.

File: 25-evaluation-3/PRESENTATION.md (hunk 2 above)
Comment (ludelafo) [reply 25-evaluation-3/PRESENTATION.md:125]: The same as my previous comment: 1 sheet from the first evaluation, 1.5 from the second and 1.5 new sheets of paper for the last evaluation, a total of 4 sheets for the exam.

**Pull request merged**